docs/source/sensor.camera.rst

```diff
@@ -0,0 +1,20 @@
+kornia.sensor.camera
+======================
+
+.. currentmodule:: kornia.sensor.camera
+
+Camera models available in Kornia are as follows:
+
+#. Pinhole.
+#. Brown Conrady.
+#. Kannala Brandt K3.
+#. Orthographic.
```

Comment:
We could also add some extra documentation to explain the usefulness and differences of each -- not now, we can open an issue to do it later if you agree

Reply:
yes, have only written minimally here.